### DISCUSSION TITLE
Use with diaphragm gas meter?

### DISCUSSION THREAD

**Discussion by Haifirion** [jomjol/AI-on-the-edge-device#624] [Ideas]
Would it be possible to modify AI on the edge to be usable with a diaphragm gas meter?
The display consists of several rollers, pretty similar to the water meter, but without analog dials. So just a string of digits.
It would make the statistic data way more accurate than my current solution, through magnetic pulse counting with a hall sensor.
Best regards 
Haifi

**Comment by jomjol** [comment]
Can you post a picture?

**Reply by Haifirion** [reply]
Thx for the reply, but I figured out that this is already possible with your current version. Great job on that project by the way, and keep up the good work!
Best regards
Haifi
